packages/next/build/babel/preset.ts

```diff
@@ -182,6 +182,7 @@ module.exports = (
       require('@babel/plugin-proposal-nullish-coalescing-operator'),
       isServer && require('@babel/plugin-syntax-bigint'),
       [require('@babel/plugin-proposal-numeric-separator').default, false],
+      require('@babel/plugin-proposal-export-namespace-from'),
```

Comment:
This needs an integration test to ensure we don't regress on it later on, you can add it to `test/integration/babel`

Reply:
It's done, thank you for the tip and for guiding me throughout the whole review process!